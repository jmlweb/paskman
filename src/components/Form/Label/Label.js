// @flow
import styled from 'styled-components';
import colors from '../../../styles/colors';
import type { Target } from 'styled-components';

const Label: Target = styled.label`
  font-weight: 500;
  display: block;
  font-size: 1.4rem;
  text-transform: uppercase;
  margin-bottom: 0.5rem;
  color: ${colors.secondaryA7};
  b,
  strong {
    font-weight: 700;
  }
`;

export default Label;
