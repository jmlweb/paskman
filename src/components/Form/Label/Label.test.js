import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Label from './Label';

describe('Label', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = mount(<Label>Dummy</Label>);
  });
  it('Render the component', () => {
    expect(wrapper.find(Label).length).toEqual(1);
  });
  it('Matchs snapshot', () => {
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
