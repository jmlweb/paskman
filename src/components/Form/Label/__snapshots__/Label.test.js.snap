// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Matchs snapshot 1`] = `
<label
  className="sc-bdVaJa khRasU"
>
  Dummy
</label>
`;
